docs/distributions/gentoo/installation.md

```diff
@@ -1,89 +1,46 @@
-3. Ensure that Secure Boot is disabled
-
-    1. Follow the instructions from [this article](https://support.apple.com/en-us/HT208198)
-    2. Once in the Startup Security Utility, set secure boot to no security and enable external boot.
-
-4. Boot into the Live USB enviornment
-
-    1. Plug in the USB Flash Drive into your computer
-    2. Startup while holding the Option key, this will put you in the macOS Startup Manager
-    3. Select the orange EFI Boot option and press enter to boot into it. (If you're using a Ubuntu Live environment, then make sure to select the orange EFI Boot option all the way to the right)
-
-5. Follow the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks)
+1. Follow the [Pre-installation](https://wiki.t2linux.org/guides/preinstall) steps.
+   1. Since there is not yet a T2 Gentoo Installation ISO, you will need to use a different ISO. We recommend the [T2-Ubuntu](https://github.com/t2linux/T2-Ubuntu/releases/latest) ISO if you want a graphical environment, or the [T2-Archiso](https://github.com/t2linux/archiso-t2/releases/latest) if you don't want to have to copy wifi firmware to the livecd.
+   2. If you chose the Ubuntu ISO, follow the [WiFi Guide](https://wiki.t2linux.org/guides/wifi-bluetooth/) once booted.
 
-    1. Instead of making a new EFI partition for Gentoo, you should instead mount `/dev/nvme0n1p1` to `/mnt/gentoo/boot/efi`
-    2. Make sure that you have deleted the partition you made ahead of time, or else you'll lose important data
+2. Connect to the internet using NetworkManager in the Ubuntu ISO, or using `iwctl` in the Arch ISO.
 
-6. Continue through the Handbook until chapter "Configuring the kernel"
+3. You will need to reformat your partitions, except for the EFI partition. The other partitions will need to reformatted as described in the [Gentoo Handbook](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Disks). For the EFI system partition there will be one at `/dev/nvme0n1p1` and you can use this if you don't intend to install Windows or already have it installed. If you do intend to triple boot, refer to [this guide](https://wiki.t2linux.org/guides/windows/).
 
-    1. In order to have the Linux Kernel support your device the most, you'll have to build from a modified Linux Kernel source tree.
-    2. Git clone [the t2linux kernel source tree](https://github.com/t2linux/kernel) to `/usr/src`
-        * Make sure to git checkout the version you want. For example, if you want kernel version v5.16.17, you would checkout tag `t2-v5.16.17`.
-    3. Run these commands to symlink the kernel to `/usr/src/linux`:
+4. Follow the Gentoo Handbook from [Installing stage3](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Stage) until [Kernel Configuration and Compilation](https://wiki.gentoo.org/wiki/Handbook:AMD64/Installation/Kernel#Kernel_configuration_and_compilation).
```

Comment:
Installing stage3, I don’t get what does this mean here.

Reply:
@AdityaGarg8 I think it means the manual index "Installing stage3"
![image](https://user-images.githubusercontent.com/39993457/235346201-748df849-d4b5-4652-b16d-da394359c388.png)

Reply:
Yeah, it's referring to installing the stage 3 tarball.